velodyne_pointcloud/src/lib/rawdata.cc

```diff
@@ -305,7 +305,9 @@ inline float SQR(float val) { return val*val; }
           intensity = (intensity < min_intensity) ? min_intensity : intensity;
           intensity = (intensity > max_intensity) ? max_intensity : intensity;
   
-          data.addPoint(x_coord, y_coord, z_coord, corrections.laser_ring, raw->blocks[i].rotation, distance, intensity);
+          data.addPoint(x_coord, y_coord, z_coord, corrections.laser_ring,
+              raw->blocks[i].rotation, distance, intensity,
+              velodyne_pointcloud::ReturnType::UNKNOWN);
```

Comment:
In C++, using ReturnType:: doesn't work for an enum. A standard enum just gets it's contents declared as globals. To namespace-scope the values, use `enum class` in the declaration instead. See https://stackoverflow.com/questions/482745/namespaces-for-enum-types-best-practices

Reply:
Since this needs to build on indigo I can't use the new enum class scoping :cry: 
Reverted to using old school normal enum with implicit type casting.

Reply:
Honestly, I'm not too worried about it building on Indigo anymore. Indigo/Trusty will be EOL in April and I likely won't release a new version for Indigo with this functionality included. If you want, you can remove the Indigo build from Travis as part of this PR.